lib/loaders/loaders_with_authentication/impulse.js

```diff
@@ -21,7 +21,7 @@ export default (accessToken, userID, requestIDs) => {
       from_id: userID,
       from_type: "User",
     }),
-    conversationLoader: impulseLoader(id => `conversations/${id}`),
+    conversationLoader: impulseLoader(id => `conversations/${id}`, { has_message: true }),
```

Comment:
Nice, winds up using [this scope](https://github.com/artsy/impulse/blob/83c566553eeb8402ef33b6e7fe49ed4b7a19e2ad/app/models/conversation.rb#L65), which looks 👍